pointInPoly.ts

```diff
@@ -0,0 +1,185 @@
+// Check is a point (x,y,z) is contained within a polygon's bounding box
```

Comment:
Could you add type annotations that aren't just  `: any` to make this file easier to follow/use? You can use https://www.npmjs.com/package/@types/geojson to get GeoJSON type definitions

Reply:
Yep 👍